dotcom-rendering/src/layouts/AllEditorialNewslettersPageLayout.tsx

```diff
@@ -58,7 +58,10 @@ export const AllEditorialNewslettersPageLayout = ({
 								padSides={false}
 								shouldCenter={false}
 							>
-								<HeaderAdSlot />
+								<HeaderAdSlot
+									isPaidContent={false}
+									shouldHideReaderRevenue={false}
```

Comment:
Flagging this here as it appears it's not possible to obtain a shouldHideReaderRevenue / isPaidContent flags for an editorial newsletters page.